docs/onboarding/8_16/operator/onprem_kube_stack_values.yaml

```diff
@@ -0,0 +1,407 @@
+opentelemetry-operator:
+  manager:
+    extraArgs:
+      - --enable-go-instrumentation
+  admissionWebhooks:
+    certManager:
+      enabled: false
+
+autoGenerateCert:
+  enabled: true
+  recreate: true
+
+crds:
+  create: true
+
+defaultCRConfig:
+  image:
+    repository: "docker.elastic.co/beats/elastic-agent"
+    tag: "8.16.0-SNAPSHOT"
+  targetAllocator:
+    enabled: false
+  env:
+    - name: ELASTIC_AGENT_OTEL
+      value: '"true"'
+    - name: ELASTIC_ENDPOINT
+      valueFrom:
+        secretKeyRef:
+          name: elastic-secret-otel
+          key: elastic_endpoint
+    - name: ELASTIC_API_KEY
+      valueFrom:
+        secretKeyRef:
+          name: elastic-secret-otel
+          key: elastic_api_key
+    - name: K8S_NODE_NAME
```

Comment:
since we are relying on the `opentelemetry-kube-stack` helm chart - I think we can use `OTEL_K8S_NODE_NAME` env variables provided in the chart instead - https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-kube-stack/templates/collector.yaml#L138-L141

Reply:
Good catch, fixed in https://github.com/elastic/opentelemetry/pull/11/commits/253222c8cfb709824dc45fd6dc746d7a07783d77